### ISSUE TITLE
[Bug]: The 'sqlean' library does not work on Intel CPUs

### ISSUE THREAD

**Issue by lucydodo** [sqlitebrowser/sqlitebrowser#3647]
### What did you do?

Try to load 'sqlean' library.

### What did you expect to see?

Loaded.

### What did you see instead?

Not loaded.

### DB4S Version

3.13.99 (nightly)

### What OS are you seeing the problem on?

MacOS

### Prevention against duplicate issues

- [X] I have searched for similar issues

**Comment by lucydodo** [comment]
I'm writing this in case anyone else runs into the sasme problem.

Currently, the 'sqlean' library included in macOS builds is only signed, packaged and distributed as an ARM64 binary. I did this and forgot to include the x86_64 binaries. I realized this a while ago, and I just happened to be working on a patch internally that made some changes to our CI system for the next release, so I patched this issue as well. and I'll be pushing this patch upstream in the near future and will fix this issue as well.

## If you need to load the 'sqlean' library on an Intel CPU before the patch is pushed, please note the following:

1. Download the library for x86_64 processors from the 'sqlean' the [official website.](https://github.com/nalgeon/sqlean)
2. Run the following command in a terminal
```bash
codesign --force --deep -s - /Applications/DB\ Browser\ for\ SQLite.app
codesign --force -s - ~/Downloads/sqlean.dylib
```

### What does this command do, and why do I need to do it?
> 'DB Browser for SQLite' is code-signed and notarized by Apple to ensure the integrity of the binary. As part of this "notarization" process, Apple requires that external libaries that the application loads also receive the same code signature as the DB4S team. Therefore, external libraries which are not signed by the DB4S team cannot be imported, so you can change them to be imported with a temporary signature, an 'ad-hoc signature', using the command above.

If you have any additional questions, please leave a comment on this issue and I'll be happy to answer them.

**Comment by lucydodo** [comment]
Resolved via 510cfbc11377be11dfa6601b7942ac469530d644
